app/crontask.go

```diff
@@ -0,0 +1,51 @@
+package app
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave-starter-kit/x/custom"
+)
+
+// CronTaskMarshaler is a task marshaler implementation to be used by the bnsd
+// application when dealing with scheduled tasks.
+//
+// This implementation relies on the CronTask protobuf declaration.
+var CronTaskMarshaler = taskMarshaler{}
+
+type taskMarshaler struct{}
+
+// MarshalTask implements cron.TaskMarshaler interface.
+func (taskMarshaler) MarshalTask(auth []weave.Condition, msg weave.Msg) ([]byte, error) {
+	t := CronTask{
+		Authenticators: auth,
+	}
+
+	switch msg := msg.(type) {
+	default:
+		return nil, errors.Wrapf(errors.ErrType, "unsupported message type: %T", msg)
+
+		case *custom.DeleteTimedStateMsg:
```

Comment:
Please run `go fmt` on this file.

Reply:
Could not make vscode auto fmt work. Fixed the issue with using a githook.